### PR TITLE
New Site Information

### PR DESCRIPTION
**Pull request by Frazl**
Added basic documentation regarding the implementation of the new rb site.

### CODE REVIEW

**Review by theycallmemac — COMMENTED** [review]
I'm gunna be that guy - it should be: "We use an apache macro template for user vhosts."

File: docs/web/apache.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ other machines use 2.4. See [Apache Modules](/web/apachemodules) for modules.
 
 We use a apache macro template for user vhosts.
 
+## www.redbrick.dcu.ie
+
+The config file for www.redbrick.dcu.ie can be found on Metharme.
+
+`
```
Comment (butlerx) [inline docs/web/apache.md:14]: Tripple ticks

File: docs/web/website.md [hunk 2]
```diff
@@ -1,77 +1,72 @@
-# Static-Site
+# Redbrick Site
```
Comment (butlerx) [inline docs/web/website.md:1]: trailing whitespace

File: docs/web/website.md [hunk 3]
```diff
@@ -1,77 +1,72 @@
...
+## Google Sheet 
 
-1. **Node** : v6.9(LTS) Download [Node](https://nodejs.org/download/)
+The google sheet controlling the information can be found [here](https://docs.google.com/spreadsheets/d/15pFYVzuFPK4HFOpnnFHuoTNh3L3iThBCpoVMQzT5RlM/edit?usp=sharing).
 
-## Setup
+## Data Endpoint 
 
-To set up run:
+The events, images, and slides get pulled from the root directory of 'fraz'. This file gets updated every 24 hours at 00:00 from the google sheet. The google sheet can only be edited by cmt members.
```
Comment (butlerx) [inline docs/web/website.md:14]: are you using your account to build this?

File: docs/web/website.md [hunk 4]
```diff
@@ -1,77 +1,72 @@
...
-- `email_update_log` a newline-separated reverse-chronological list of times
-  email updates were sent
-- `.env` by copying `.env.example` and **modifying values (_important_)**
+1. git clone the repo.
+2. run npm install.
+3. ensure the "homepage" in package.json is set correctly.
+4. ensure this.dataURL is set to pull information from the correct JSON file. 
+5. run npm build.
+6. ensure there are no errors in the output.
+7. copy &amp; paste the .htaccess file located in the apache folder into the build folder.
```
Comment (butlerx) [inline docs/web/website.md:29]: you can use `&` in md rather then `&amp;`

File: docs/web/apache.md (hunk 1 above)
Comment (Frazl) [reply docs/web/apache.md:14]: You said in the last PR #127 to you'd prefer single ticks...

File: docs/web/website.md (hunk 3 above)
Comment (Frazl) [reply docs/web/website.md:14]: Yes this is currently running off my account as it's using my GCP credit. Looking at alternatives to store this information and edit it easily still.

**Comment by Frazl** [comment]
> I'm gunna be that guy - it should be: "We use an apache macro template for user vhosts."

I didn't touch what was already there and this isn't concerned with this PR 👍

File: docs/web/apache.md (hunk 1 above)
Comment (butlerx) [reply docs/web/apache.md:14]: well this is embarrassing.
the real issue I have is when using single ticks the content should be on the same line as the ticket. 

I realised it's not required by the repo or standards but using something like prettier(which I used when porting to mkdocs and updating the docs) would remove most of the stylist points.

File: docs/web/website.md (hunk 3 above)
Comment (butlerx) [reply docs/web/website.md:14]: this is in GCP?

**Review by theycallmemac — APPROVED** [review]

**Pull request merged**